src/momento/responses/response.py

```diff
@@ -0,0 +1,5 @@
+from abc import ABC
+
+
+class CacheResponse(ABC):
```

Comment:
Can you add `CacheResponse` to the init file under `src/momento/responses`? That will allow us type `from momento.responses import CacheResponse` instead of `from momento.responses.response import CacheResponse`. 

Other than that looks good